### PR TITLE
RSVP FE preview work for G20.06

### PR DESCRIPTION
**Pull request by sc0ttkclark**
Holding on merge for kitchen sink, code reviewed already from PRs to this bucket branch.

### CODE REVIEW

**Comment by juanfra** [comment]
@sc0ttkclark should we include a readme entry for the notice?

**Comment by sc0ttkclark** [comment]
The notice is disabled right now, correct?

File: src/Tribe/Admin/Notices.php [hunk 1]
```diff
@@ -13,7 +13,102 @@ class Tribe__Tickets__Admin__Notices {
...
+			! function_exists( 'tribe_installed_before' )
+			|| ! class_exists( 'Tribe__Admin__Notices' )
+		) {
+			return;
+		}
+
+		$this->maybe_display_plus_commerce_notice();
+
+		// @todo Uncomment this when we are ready to show the opt-in notice in G20.07.
+		// $this->maybe_display_rsvp_new_views_options_notice();
```
Comment (sc0ttkclark) [inline src/Tribe/Admin/Notices.php:36]: RSVP notice disabled here @juanfra

File: src/Tribe/Admin/Display_Settings.php [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	 *
+	 * @since TBD
+	 *
+	 * @param array $settings List of display settings.
+	 *
+	 * @return array List of display settings.
+	 */
+	public function add_display_settings( $settings ) {
+		// @todo Remove this in G20.07
+		if ( null === constant( 'TRIBE_TICKETS_RSVP_NEW_VIEWS' ) ) {
```
Comment (sc0ttkclark) [inline src/Tribe/Admin/Display_Settings.php:30]: The admin option only shows if you have the constant manually set

Comment (juanfra) [reply src/Tribe/Admin/Display_Settings.php:30]: oh right! forgot that

**Pull request merged**